### PR TITLE
Afid flip

### PR DESCRIPTION
**Pull request by kaitj**
## Proposed changes

Checks version number of Slicer and performs flipping of fiducials from RAS to LPS (if v4.11+). This change addresses #72, where coordinate flips were noticed. Fix is to multiply the x and y coordinates of each fiducial.

Additionally adds unit test for a `valid_flip.fcsv` file.

## Types of changes

What types of changes does your code introduce? Put an `x` in the boxes that apply

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionalitiy)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (if none of the other choices apply)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [x] Changes have been tested to ensure that fix is effective or that a feature works.
- [x] Changes pass the unit tests
- [x] I have included necessary documentation or comments (as necessary)
- [x] Any dependent changes have been merged and published

## Notes
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

_PR template was adopted from [appium](https://github.com/appium/appium/blob/master/.github/PULL_REQUEST_TEMPLATE.md)_

### CODE REVIEW

File: test/test_model_auto.py [hunk 1]
```diff
@@ -6,6 +6,10 @@ def test_valid(self):
         with open('test/resources/valid.fcsv', 'r') as fcsv:
             fcsv_json = model_auto.csv_to_json(fcsv)
 
+    def test_valid_flip(self):
```
Comment (tkkuehn) [inline test/test_model_auto.py:9]: Would like to test here that the coordinates actually get properly transformed to the new coordinate space in the output json.

Comment (kaitj) [reply test/test_model_auto.py:10]: Added in assertions to test for equality for x and y flipping. This is done just for the first x and y coordinate comparing against the `valid_flip.fcsv`

There may be a better way to write this?

Comment (tkkuehn) [reply test/test_model_auto.py:10]: This sounds reasonable to me!

**Comment by kaitj** [comment]
Conflcit should be resolved now. We were previously just returning `row[key]` when we weren't checking for flips. Now set up to store the `row[key]` to variable `value` in all cases (original vs flipped)

**Pull request merged**